test/src/main/java/org/apache/accumulo/test/TableOperationsIT.java

```diff
@@ -205,6 +205,17 @@ public void createTableWithTableNameLengthLimit()
     assertFalse(tableOps.exists(t2));
   }
 
+  @Test
+  public void createTableWithBadProperties()
+      throws AccumuloException, AccumuloSecurityException, TableExistsException {
+    TableOperations tableOps = accumuloClient.tableOperations();
+    String t0 = StringUtils.repeat('a', MAX_TABLE_NAME_LEN - 1);
```

Comment:
We typically create table names with something like `String t0 = getUniqueNames(1)[0];

Reply:
Ah I see that on line 221. 
So does this need to change?

Reply:
It would be nice for consistency - and the getUniqueNames uses the test and the function in the name to allow multiple tests in the same file to avoid collisions.  Not sure if it would apply here, but that would also pop up in the logs if anyone was trying to chase something down.

Reply:
Sorry @EdColeman  - I think I was typing and didn't see your response.

Reply:
Ok, I changed it and I'm running the `TableOperationsIT` now. Once it's done I'll push the commit up.